packages/eslint-config-custom/index.js

```diff
@@ -27,7 +27,7 @@ module.exports = {
     ],
 
     'import/order': [
-      'error',
+      'warn',
```

Comment:
what's the issue here?

Reply:
There is no reason to stop the build, the preview and the CI tests just because of import order

Reply:
The warns are pretty clear on the CI run, they are fixed eventually